conda-forge.yml

```diff
@@ -1,6 +1,8 @@
+bot:
+  inspection: update-grayskull
```

Comment:
Included Grayskull for updates to pick up dependencies as noted here: https://github.com/conda-forge/sphinxcontrib-bibtex-feedstock/pull/25#issuecomment-1693794191